cc_plugin_imos/imos.py

```diff
@@ -647,8 +638,10 @@ def check_longitude_variable(self, dataset):
             valid_min = getattr(longitude_var, 'valid_min', None)
             valid_max = getattr(longitude_var, 'valid_max', None)
             if (valid_min, valid_max) not in valid_range_expected:
-                result = Result(BaseCheck.HIGH, False, name=('var', 'LONGITUDE', 'valid_min/max'))
-                result.msgs = ['(valid_min, valid_max) should be %s or %s' % valid_range_expected]
+                result = Result(BaseCheck.HIGH, False, name='LONGITUDE')
+                result.msgs = [
+                    'Attributes (valid_min, valid_max) should be %s or %s.' % valid_range_expected
+                ]
```

Comment:
Actually it is being done. For LATITUDE it's a simpler check because there's only one right answer (-90, 90), which can be specified in the `latitude_attributes` dict (https://github.com/aodn/cc-plugin-imos/blob/master/cc_plugin_imos/imos.py#L661).